query.go

```diff
@@ -792,7 +792,8 @@ func (qm *QueryMatch) Id() uint {
 func newQueryMatch(m *C.TSQueryMatch, cursor *C.TSQueryCursor) QueryMatch {
 	var captures []QueryCapture
 	if m.capture_count > 0 {
-		captures = (*[1 << 16]QueryCapture)(unsafe.Pointer(m.captures))[:m.capture_count:m.capture_count]
+		cc := unsafe.Slice(m.captures, m.capture_count)
+		captures = *(*[]QueryCapture)(unsafe.Pointer(&cc))
```

Comment:
```suggestion
		captures = *(*[]QueryCapture)(unsafe.Pointer(&unsafe.Slice(m.captures, m.capture_count)))
```

Reply:
annoyingly this does not work in go

Reply:
ah okay that's fine. let's just rename `cc` to `c_captures` then, `cc` tells us nothing

Reply:
renamed it to `cCaptures` since go naming conventions